packages/miniflare/src/plugins/core/index.ts

```diff
@@ -118,6 +118,7 @@ const UnusableStringSchema = z.string().transform(() => undefined);
 export const UnsafeDirectSocketSchema = z.object({
 	host: z.ostring(),
 	port: z.onumber(),
+	serviceName: z.ostring(),
```

Comment:
The `serviceName` option will enable us to register a worker in the dev registry with the name of the current worker but pointed to a different service (i.e. The PRC Proxy worker in this case)